CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 CMAKE_MINIMUM_REQUIRED (VERSION 3.5.0)
-CMAKE_POLICY(SET CMP0148 OLD)
+CMAKE_POLICY(VERSION 3.26.5)
```

Comment:
CMAKE_POLICY minimal version must not exceed currently running cmake. I.e. this change causes a failure if running on cmake older than 3.26.5. (Try increasing the value on 3.27.5 and running it on CentOS 9.) This change effectively sets a minimal cmake version to 3.26.5. From the CMake documentation:

>  The `min` version must be at least 2.4 and at most the running version of CMake.

If you want to do so, then also update dnf.spec to enforce cmake ≥ 3.26.5. But then why not simply increasing CMAKE_MINIMUM_REQUIRED() to 3.26.5 a line above?

However, I have a different proposal: Remove the CMAKE_POLICY() line. What benefit does it bring? That hosts's cmake behaves like cmake of the given version everywhere?